sql/telemetry_derived/clients_daily_keyed_histogram_aggregates_v1/query.sql

```diff
@@ -85,16 +85,28 @@ WITH filtered AS (
         )
         AND client_id IS NOT NULL),
 
+build_filtered AS (
+    SELECT
+      client_id,
+      submission_date,
+      os,
+      app_version,
+      app_build_id,
+      payload,
+      COALESCE(glam_valid_builds_v1.channel, channel_filtered.channel) AS channel
+    FROM channel_filtered
+    LEFT JOIN glam_valid_builds_v1
+      USING (channel)
+    WHERE app_build_id IN UNNEST(build_ids)),
```

Comment:
If this were a flat dataset, it would just be the inner join of the `channel_filter` and `glam_valid_builds_v1` using `channel` and `build_id`.